x/rollup/module.go

```diff
@@ -57,7 +60,13 @@ func ProvideCustomGetSigner() signing.CustomGetSigner {
 }
 
 func ProvideModule(in ModuleInputs) ModuleOutputs { //nolint:gocritic // hugeParam
-	k := keeper.NewKeeper(in.Codec, in.StoreService, in.BankKeeper, in.AccountKeeper)
+	// default to governance authority if not provided
+	authority := authtypes.NewModuleAddress(govtypes.ModuleName)
+	if in.Config.Authority != "" {
+		authority = authtypes.NewModuleAddressOrBech32Address(in.Config.Authority)
+	}
```

Comment:
Note for reviewer: this follows the same flow as the OOTB cosmos sdk modules.

[x/bank example](https://github.com/cosmos/cosmos-sdk/blob/v0.50.9/x/bank/module.go#L243-L247)

Reply:
ootb?